### PR TITLE
[SDG/Explore] Some UI updates

### PR DESCRIPTION
**Pull request by chejennifer**
- add link to place explorer on place overview tile
Screenshot 2023-09-10 at 10 38 43 AM

- update sdg homepage place section according to mocks
Screenshot 2023-09-10 at 10 39 00 AM

### CODE REVIEW

File: static/css/shared/_tiles.scss [hunk 1]
```diff
@@ -94,7 +94,7 @@ $box-shadow-8dp: 0 8px 10px 1px rgba(0, 0, 0, 0.14),
 .chart-container.line-chart,
 .chart-container.bar-chart,
 .chart-container.histogram-chart,
-.chart-container.disaster-event-map-tile {
+.chart-container.disaster-event-map-tile{
```
Comment (juliawu) [inline static/css/shared/_tiles.scss:97]: super nit: add space back?

**Review by juliawu — APPROVED** [review]
Nice!

File: static/css/shared/_tiles.scss (hunk 1 above)
Comment (chejennifer) [reply static/css/shared/_tiles.scss]: done

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**